src/core/agent/agent.ts

```diff
@@ -174,48 +178,54 @@ class Agent {
     );
   }
 
-  async start(agentUrls: Partial<AgentUrls>): Promise<void> {
+  async start(keriUrl?: string): Promise<void> {
```

Comment:
This doesn't need this parameter - it can just read from the DB

Reply:
Actually, lets change this. Lets make it so that it only accepts a connect URL and that the AppWrapper must fetch it. So we can remove `getAgentUrls` from here. And in the AppWrapper:
- If it's stored, call connect
- If not, we will need to make sure we are on the onboarding screen to enter it